src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Current.net45.cs

```diff
@@ -8,6 +8,7 @@
 
 namespace System.Diagnostics
 {
+    [Serializable]
```

Comment:
Can you add a test for this?

And is there any value in making serializable everywhere, not just 4.5, if for no other reason than consistency?